pennylane/ops/qubit/hamiltonian.py

```diff
@@ -323,7 +332,7 @@ def simplify(self):
         **Example**
 
         >>> ops = [qml.PauliY(2), qml.PauliX(0) @ qml.Identity(1), qml.PauliX(0)]
-        >>> H = qml.Hamiltonian([1, 1, -2], ops)
+        >>> H = qml.Hamiltonian([1, 1, -2], new_ops)
```

Comment:
how come this changed? It looks like the original `ops` was correct?

Reply:
Great catch, that was a name refactor side effect!